### ISSUE TITLE
Persistent data at Azure

### ISSUE THREAD

**Issue by antoniovj1** [sverhoeven/docker-cartodb#50]
I'm able to deploy the container to Azure, and everything seems to be running ok. But I'm trying to make data persistent and I'm unable of running the comands of the documentation since Azure limitations. Is there other way to make data persistent? For example inside the container.

Thanks

**Comment by rkertesz** [comment]
Is this related to issue #51 that I posted, Or are you asking how to use azure file share with a container? (In my case, I am using an aci) .

**Comment by sverhoeven** [comment]
In the fix for #51 the chown happens during container startup. 

The startup script could be extended that when the postgresql dir is empty it runs
```
# start postgresql server  + redis
# initdb 
# install the pg extensions?
cd /cartodb
script/create_dev_user
script/setup_organization.sh
script/geocoder.sh
```
Not sure about the first commands as these are scattered in the Dockerfile.

In Azure when an empty volume is mounted at /var/lib/postgresql is should be filled during the startup once and later container restarts should persist data on the volume.
